### DISCUSSION TITLE
Volume Control for Windows App in Windows 10 store

### DISCUSSION THREAD

**Discussion by Kavboy** [File-New-Project/EarTrumpet#815] [General]
I found this app in the store today.
https://www.microsoft.com/de-de/p/volume-control-for-windows/9nfhvp7hnhc9?cid=msft_web_chart&activetab=pivot:overviewtab#

The screenshots look startingly like EarTrumpet.

I didn´t install it, because it gains all permissions and I don´t trust it. 
So just wanted to let you know about it.

**Comment by riverar** [comment]
Thanks @Kavboy. I sent a few emails. While our code is licensed in a way that permits this (MIT), the publisher failed to abide by the single condition MIT requires, and claims copyright of our code. 🤦‍♂️
